leakcanary-android-core/src/main/java/leakcanary/internal/activity/LeakActivity.kt

```diff
@@ -20,6 +20,7 @@ import shark.SharkLog
 import java.io.FileInputStream
 import java.io.IOException
 
+
 internal class LeakActivity : NavigatingActivity() {
```

Comment:
welcome to android 🤷‍♂️

Some feedback: LCOM should be linked to a definition so that people understand what it's about. Also `0.8611111111111112` is not useful, `0.9` would be enough here no?

Reply:
Thank you for leaving some feedback :)! I agree.